docs/index.rst

```diff
@@ -18,21 +18,33 @@ Overview
 
    install
 
+.. _tasks_toc:
+
+Tasks
+=====
+
+At the core of Scintillometry is a collection of "tasks", filehandle-like
+classes that can be linked together into a data reduction pipeline.
+
+.. toctree::
+   :maxdepth: 1
+
+   tasks/channelize
+   tasks/base
+
 .. _modules_toc:
 
-Modules
-=======
+Support Modules
+===============
 
-Scintillometry is divided into modules, each of which can be used standalone
-given correct inputs.
+Scintillometry also contains support modules that assist with calculations and
+that link it with other software, such as Fourier transform packages.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-   base/index
-   channelize/index
-   dispersion/index
-   fourier/index
+   supportmodules/dm
```

Comment:
In analogy with `baseband`, let's just call the directory `helpers`

Reply:
Sure, changed.